automation/taskcluster/lib/tasks.py

```diff
@@ -80,7 +84,7 @@ def craft_assemble_nightly_task(self, variant, version_name, is_staging):
             description='Building and testing variant {}'.format(variant.name),
             command=command,
             scopes=[
-                "secrets:get:{}".format(sentry_secret)
+                "secrets:get:{}".format(secret_index)
```

Comment:
the scope here seems to be not correct. you will need a scope for every secret.

Reply:
IIRC, if you follow what Fenix does, you should have a single secret for `reference-browser/nightly` :)